carousel/tests/test_core.py

```diff
@@ -15,3 +15,14 @@ def test_pv_context():
     esun = Q_(0.8765, UREG.suns)
     ok_(esun.dimensionless)
     eq_(esun.to('W / m ** 2', 'pv'), 876.5 * UREG.W / UREG.m / UREG.m)
+
+
+def test_fields():
+    """
+    Test that Carousel field creates an object with attributes
+    """
+    Field._attrs = ['units', 'isconstant']
+    test_field = Field('my test field', units='W/m**2', isconstant=True)
```

Comment:
so here 'my test field' will be assigned to the 'units' attribute first? But then replaced by 'W/m**2' it looks like, is that correct?